speechbrain/utils/logger.py

```diff
@@ -100,7 +100,7 @@ def setup_logging(
         recursive_update(config, overrides)
         logging.config.dictConfig(config)
     else:
-        logging.basicConfig(level=default_level)
+        logging.basicConfig(level=default_level, format='%(asctime)s | %(module)s-%(lineno)d | %(levelname)s | %(message)s')
```

Comment:
Why is this needed ?

Reply:
By default the output from a message "bla bla" will look like:
```
INFO:root:bla bla
```
With this change a lot more useful information (notice it also prints the module name and line number) is added to the output, now it looks like:
```
2022-04-05 12:14:15,431 | test-4 | INFO | bla bla
```